Doc/library/smtpd.rst

```diff
@@ -148,6 +148,12 @@ MailmanProxy Objects
 
 .. class:: MailmanProxy(localaddr, remoteaddr)
 
+   .. deprecated-removed:: 3.8 3.10
+
+      :class:`MailmanProxy` is deprecated, it depends on a ``Mailman``
+      module which no long exists and therefore is already broken.
```

Comment:
no longer

Reply:
Done. By the way, github suggestions make tiny fixes like this very easy to suggest and accept.